src/actions/config.js

```diff
@@ -44,35 +49,54 @@ export function asyncConfigInit() {
   };
 }
 
-export function asyncConfigFetch(configUrlList) {
+export function asyncConfigFetch() {
   return dispatch => {
     dispatch(configFetch());
     let opts = {};
     opts.headers = { Accept: 'application/javascript' };
-    let configUrl = configUrlList.shift();
+    let configUrl = './config.js';
     http.get(configUrl, opts, function(error, response) {
       if (response) {
         dispatch(configFetchSuccess());
         try {
-          let userConfig = JSON.parse(response.text);
-          Object.keys(userConfig).map(function(key) {
-            dispatch(configUpdateByKey(key, userConfig[key]));
+          let config = JSON.parse(response.text);
+          Object.keys(config).map(function(key) {
+            dispatch(configUpdateByKey(key, config[key]));
           });
-          if (configUrlList.length > 0) {
-            dispatch(asyncConfigFetch(configUrlList));
-          } else {
-            dispatch(asyncIntlFetchTranslations());
-          }
         } catch (e) {
           dispatch(notificationAddError(e.message + ' ' + configUrl));
         }
+        dispatch(asyncUserConfigFetch());
       } else {
         dispatch(configFetchError());
       }
     });
   };
 }
 
+export function asyncUserConfigFetch() {
+  return dispatch => {
+    dispatch(configFetch());
+    let opts = {};
+    opts.headers = { Accept: 'application/javascript' };
+    let configUrl = './userConfig.js';
```

Comment:
From [MIME type list](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Complete_list_of_MIME_types):

We've only 2 options: CSV and XML. Both are really ugly. I don't think we can create our own type or use YAML since the firewalls will block those as well.